lib/LaTeXML/Core/Mouth.pm

```diff
@@ -212,7 +212,7 @@ sub handle_EOL {
   # but it makes nicer XML with occasional \n. Hopefully, this is harmless?
   my $token = ($$self{colno} == 1
     ? T_CS('\par')
-    : ($STATE->lookupValue('PRESERVE_NEWLINES') ? Token("\n", CC_SPACE) : T_SPACE));
+    : (($STATE->lookupValue('PRESERVE_NEWLINES') || 0) == 1 ? Token("\n", CC_SPACE) : T_SPACE));
```

Comment:
This doesn't seem to be needed (or wanted?) as far as the fidelity of `\read` is concerned. You always get catcode `CC_SPACE`, but whether it contains a space or cr. It does affect the XML output, though.

Reply:
Indeed, this exact difference aims to reduce the number of line-breaks that reach the XML in a way that is closer to TeX.

So in `io.pdf` you will see:
> Line: “This file exists! ”

and

> Line: “This file exists!fl”

while in the `io.xml` you will see them [with line breaks](https://github.com/brucemiller/LaTeXML/pull/1429/files#diff-22a41af1c2e122c91da0fc36220d5200299d7dda29b9058430bdd29d10d58db9), without this change.

Reply:
@brucemiller are we discussing the comment above that I exemplified with io.pdf? Just making sure I'm not misunderstanding the point.